frontend/src/components/QuizQuestion.vue

```diff
@@ -1,10 +1,10 @@
 <script lang="ts" setup>
```

Comment:
This file is starting to get a bit big, nothing to worry about right now. But we could probably refactor this later.